libtiledbvcf/src/utils/utils.cc

```diff
@@ -582,6 +583,18 @@ uint32_t system_memory_mb() {
 #endif
 }
 
+std::string memory_usage_str() {
+  std::string filename = "/proc/self/statm";
```

Comment:
Does this need to be limited to linux?

Reply:
Yes, it's a good idea to explicitly limit this code so it only tries to open the `statm` file on linux.